ChangeLog

```diff
@@ -1,3 +1,10 @@
+Version 17.13.0
```

Comment:
Add https://github.com/thought-machine/please/commit/46c31129d248f875017715f35196ee540ea049c8 and https://github.com/thought-machine/please/commit/b456d7c2ce3be061feee4aa429a810c9fd575590 here too?